packages/react-composites/tests/browser/chat/fake-adapter/fixture.ts

```diff
@@ -36,15 +36,17 @@ const buildUrlForChatAppUsingFakeAdapter = (
   return url;
 };
 
+/**
+ * Base chat adapter model
+ */
+export const BASE_CHAT_ADAPTER_MODEL = {
```

Comment:
nit: This isn't really a model for the adapter. The fake adapter uses realistic state internally.

These are just `FAKE_CHAT_ADAPTER_ARGS` that tell us how to construct the fake.